apps/frontend/src/modules/workspace/components/invite/WorkspaceInvite.tsx

```diff
@@ -124,16 +124,12 @@ export const WorkspaceInvite = ({ open, onOpenChange }: Props) => {
         }
     };
 
-    if (loadingEmailInvite) {
-        return <GlobalSpinner />;
-    }
-
     return (
         <Dialog open={open} onOpenChange={onOpenChange}>
             <DialogContent
                 alignHeader="left"
                 title={`Invite others to '${workspace?.name}'`}
-                description="Invite others to your project to collaborate together in intergraflow. An invite is specific to an email address and expires after 3 days."
+                description="Invite others to your project to collaborate together in integraflow. An invite is specific to an email address and expires after 3 days."
```

Comment:
@afolasope Can you check all occurrences of `integraflow` and then change them to `Integraflow`.